app/assets/images/payment_icons/gmo-postpay.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-gmo-postpay">
+  <title id="pi-gmo-postpay">GMO後払い</title>
+  <g>
+    <g>
+      <path fill="#000" opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/>
+      <path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/>
+    </g>
+    <g>
+      <path fill="#005bac" d="M34.12,13.5c-.63,0-.76-.79-.76-1.45a9.15,9.15,0,0,1,.26-2.12v0h-.72v0A9.92,9.92,0,0,0,32.66,12c0,1.88.87,2.15,1.39,2.15a2.42,2.42,0,0,0,1.32-.49h0v-.78l-.05,0A2.15,2.15,0,0,1,34.12,13.5Z"/>
+      <path fill="#005bac" d="M36.54,10h-.83l0,.05a5.13,5.13,0,0,1,1.08,3.55v0h.73v0A5.57,5.57,0,0,0,36.54,10Z"/>
+      <path fill="#005bac" d="M23.6,9.88v0h-.69v0a1.54,1.54,0,0,1-.92.93h0v.66h0A2.46,2.46,0,0,0,23.6,9.88Z"/>
+      <path fill="#005bac" d="M25.29,12a2.09,2.09,0,0,0,.83-.15l.06.23h.59l-.31-1.26h-.62l.15.54a2.06,2.06,0,0,1-.7.12h-.44l1.6-1.3H25.6l-.51.4h-.62l.82-.69H24.5l-.86.71V11h.85l-.52.41h-.51a1.82,1.82,0,0,0,.11-.3v0h-.65v0A1.75,1.75,0,0,1,22,12.23h0v.65h0a2.71,2.71,0,0,0,.52-.29v1.57h.62V12.09h-.09a3.47,3.47,0,0,0,.34-.54V12h.77a1.38,1.38,0,0,1-.75.66h0v.61h0A3.52,3.52,0,0,0,23.9,13a2.91,2.91,0,0,0,.47.46,4.64,4.64,0,0,1-.93.12h0v.54h0a5.35,5.35,0,0,0,1.6-.29,4.88,4.88,0,0,0,1.63.29h0v-.54h0a5.53,5.53,0,0,1-.94-.1,2.07,2.07,0,0,0,.81-.83h0v-.46H24.72a1.66,1.66,0,0,0,.13-.27Zm.53.76a2.35,2.35,0,0,1-.81.57,1.3,1.3,0,0,1-.52-.42v0h-.29l.15-.13Z"/>
+      <path fill="#005bac" d="M31.08,11.92l.3,1.3a1.69,1.69,0,0,1-1,.26h-.22l.66-3.64h-.7l-.63,3.64h-.27v.62h1.13a2.14,2.14,0,0,0,1.14-.27l.07.33h.63l-.47-2.24Z"/>
+      <path fill="#005bac" d="M29.42,12.4v-.62l-.05,0-.53.16V11h.58v-.57h-.58V9.84h-.67v.55h-.7V11h.7V12.1l-.67.08h0v.6h0l.66-.09v.75a.15.15,0,0,1-.16.13H27.6v.59h.62c.39,0,.62-.21.62-.55v-1c.23-.05.4-.1.56-.15Z"/>
+    </g>
+    <g>
+      <path fill="#005bac" d="M10.83,13.82h.28s0,0,0,0l1.17-1.91L13,14.07l0,0h1.22s0,0,0,0L12.94,10l0,0h-.46l0,0-1.58,2.44L9.26,10l0,0H8.76l0,0-1.3,4.09,0,0H8.68l0,0,.63-2.19,1.17,1.91a.05.05,0,0,0,0,0Z"/>
+      <path fill="#005bac" d="M7.49,11.78h0s0,0,0,0H4a.34.34,0,1,0,0,.67H6.05A2.16,2.16,0,0,1,4,13.56c-1.19,0-2.15-.69-2.15-1.54s1-1.53,2.15-1.53A2.62,2.62,0,0,1,5.6,11H7a0,0,0,0,0,0,0h0A4.35,4.35,0,0,0,4,9.9C2,9.9.44,10.85.44,12S2,14.15,4,14.15s3.53-1,3.53-2.13A1,1,0,0,0,7.49,11.78Z"/>
+      <path fill="#005bac" d="M17.69,9.9c-2,0-3.54.95-3.54,2.12s1.58,2.13,3.54,2.13,3.54-1,3.54-2.13S19.64,9.9,17.69,9.9Zm0,3.66c-1.19,0-2.15-.69-2.15-1.54s1-1.53,2.15-1.53,2.15.69,2.15,1.53S18.87,13.56,17.69,13.56Z"/>
+    </g>
+  </g>
+</svg>

```

Comment:
Can we optimize this SVG with `svgo` according to the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/8cc69fcbdf8c73e0ccef9010982587227825bf4f/CONTRIBUTING.md)?

```
npm install -g svgo@1.3.2
svgo app/assets/images/payment_icons/gmo-postpay.svg --disable={removeUnknownsAndDefaults,removeTitle,cleanupIDs,removeViewBox}
```